practice-one/exercise-2/part-4/js/script.js

```diff
@@ -0,0 +1,16 @@
+const getAttributes = () => {
+  const attributesHref = document.getElementById('information').href;
+  const attributesTarget = document.getElementById('information').target;
+  const attributesRel = document.getElementById('information').rel;
+  const attributesHreflang = document.getElementById('information').hreflang;
+  const attributesType = document.getElementById('information').type;
+
+  alert('designated link: ' + attributesHref);
```

Comment:
Alert should be show description that corresponding to content. How do you know which alert show content for attributesRel??